main.tf

```diff
@@ -1,9 +1,14 @@
 terraform {
+  required_version = ">= 1.0.0"
```

Comment:
I used it because it is declared in our docs that the 1.0.0 version is stable. I know that we use higher version but since this is first stable version I used it as bare minimum for version requirement.